app_base.htm

```diff
@@ -0,0 +1,83 @@
+<%inherit file="/homepage/templates/base.htm" />
+<%! from catalog import models as cmod %>
+
+<%block name="navbar">
+<header>

```

Comment:
I would indent this header tag and everything inside the block to made it easier to read.

Reply:
Good call. I'll review this section as well as my other code for instances of this mistake. Thanks!